src/main/scala/cognite/spark/v1/FlexibleDataModelsRelation.scala

```diff
@@ -203,7 +203,10 @@ class FlexibleDataModelsRelation(
     if (config.collectMetrics) {
       itemsRead.inc()
     }
-    new GenericRow(a.properties)
+    new GenericRow(a.properties.map {
+      case a: Array[Any] => new GenericRow(a)
+      case e => e
```

Comment:
It was not clear for me why do we need this?
Can you add a comment here to explain it? If waiting for the build again is too much we can "by pass" with a comment here instead but this change is weird for me

Reply:
its not bypassing.
`startNode`, `endNode` & `type` are Structs. So they are Arrays of values, which needs to be a GenericRow it self.
This function is called `toRow` and this is how it supposed to create `Row`s.
So there's nothing special here I guess. 🤔